CONTRIBUTING.md

```diff
@@ -63,119 +75,131 @@ This will open your documentation in a new browser tab at a local address (typic
+3. **Testing Your Changes**: Before finalizing your new section, it's advisable to test your changes locally. You can build or render the documentation to ensure your new section appears as expected within the documentation structure.
+  
+4. **Submitting Your Contribution**:
+Once you have created or edited the new section and verified that it meets our project's requirements, you can submit your contribution. This typically involves creating a `pull request`.
 
-2. **Updating the Sidebar**: Adding the file to the sidebar is necessary for the user to discover the content. 
-
-Contributors must add a new chapter to the sidebar. 
-
-### Adding New Sections to the Sidebar
-The sidebar in our project's documentation serves as a navigation menu, allowing users to easily access different sections. When you add a new section to the documentation, it's important to update the sidebar to include a link to the new section. This ensures that users can navigate to your content seamlessly.
-
-#### Steps:####
-1. Navigate to `_layouts/sidebar.js`
-2. Add New Section link:
-- To add a new section link, insert a new line with the following format:
-- [Intro](https://github.com/01-intro.md) -- be sure to capitalize the section title.
-- Replace "Intro" with the title of your new section.
-- Replace "/01-intro.md" with the relative path to the Markdown file of your new section.
-7. **Submitting Your Contribution**:
-Once you have created or edited the new section and verified that it meets our project's requirements, you can submit your contribution. This typically involves creating a `pull request`. 
-8. **Review and Feedback**: Be prepared to receive feedback from project maintainers or collaborators. Review and address any comments or suggestions to refine your contribution.
-9. **Documentation Completion**: After your contribution is reviewed and accepted, your new section will become part of our project's documentation, enhancing its content for users and contributors.
+5. **Review and Feedback**: Be prepared to receive feedback from project maintainers or collaborators. Review and address any comments or suggestions to refine your contribution.
+6. **Documentation Completion**: After your contribution is reviewed and accepted, your new section will become part of our project's documentation, enhancing its content for users and contributors.
 
 ## Using Markdown for Documentation
-Markdown is a lightweight and easy-to-use markup language that allows you to format text in a readable and visually appealing way. When contributing to the Intro Course, it's important to use Markdown to structure and format your content effectively.
+
+Markdown is a lightweight and easy-to-use markup language that allows you to format text in a readable and visually appealing way. When contributing to the documentation in this repository, it's important to use Markdown to structure and format your content effectively.
```

Comment:
Not a fan of how this paragraph is worded. Any suggestions on how to improve it @adiati98 and @BekahHW?

Reply:
@CBID2 what about it are you not a fan of? Can you be more specific?

Reply:
It seems a bit repetitive @BekahHW.